Week 2/Weekly Practice Problems (Week 2)/2595: Number of Even and Odd Bits/raf.py

```diff
@@ -0,0 +1,10 @@
+class Solution:
+    def evenOddBit(self, n: int) -> List[int]:
+        even, odd = 0, 0
+        binary = bin(n)[2:][::-1]
```

Comment:
This is cool!  Though we don't have to convert `n` to a binary string. We can use `%2` and `//2` to check its last bit as well!